release/models/mpls/openconfig-mpls-static.yang

```diff
@@ -217,6 +225,14 @@ submodule openconfig-mpls-static {
         "Next hop IP address for the LSP";
     }
 
+    leaf next-hop-network-instance {
```

Comment:
As some of the attributes, for ex: next-hop, push-label & etc..., in this grouping are `deprecated` wondering if this should be part of `lsp-next-hops` container.

Reply:
Yes, please move this to `lsp-next-hops` container as we are supporting adding next-hop related information in this new container only..